src/main/java/org/opentripplanner/middleware/models/OtpUser.java

```diff
@@ -86,7 +86,7 @@ public boolean delete(boolean deleteAuth0User) {
      */
     @Override
     public boolean canBeManagedBy(RequestingUser requestingUser) {
-        if (requestingUser.isThirdParty() && requestingUser.apiUser.id.equals(applicationId)) {
+        if (requestingUser.isThirdPartyUser() && requestingUser.apiUser.id.equals(applicationId)) {
```

Comment:
As mentioned above, there may be cases in the future where an ApiUser exists but is not a third party user. I don't necessarily see that happening, but I'm just not sure if it's right to include this isThirdPartyUser check here.

Reply:
Updated.